modules/core/shared/src/main/scala/gem/Program.scala

```diff
@@ -6,6 +6,8 @@ package gem
 import cats.{ Applicative, Eval, Traverse }
 import cats.implicits._
 
+import gem.syntax.treemapcompanion._
```

Comment:
Could it be folded into `gem.syntax._`?

Reply:
Yes, I could have also just imported `gem.syntax.all._`.  I just did the one since it was all I needed.